app/controllers/posts_controller.rb

```diff
@@ -33,7 +33,9 @@ def create
     respond_to do |format|
       if @post.save
         @post.create_tags_from_tag_string
-        format.html { redirect_to @post, notice: 'Post was successfully created.' }
+        format.html {
+          redirect_to post_path(@post),
+          notice: 'Post was successfully created.' }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 38, 52 should be on its own line.